source/MaterialXCore/Element.cpp

```diff
@@ -482,6 +591,70 @@ TypeDefPtr TypedElement::getTypeDef() const
 // ValueElement methods
 //
 
+bool ValueElement::isAttributeEquivalent(ConstElementPtr rhs, const string& attributeName, 
+                                         ElementEquivalenceOptions& options, ElementEquivalenceResult* result) const
+{    
+    bool perforDefaultCompare = true;
```

Comment:
What's the meaning of "perfor" that you're thinking of here?  Should we perhaps simplify this variable name to "defaultCompare"?

Reply:
Bad keyboarding. Should be `performDefaultCompare`. Could be `defaultCompare` as well.

Reply:
I've just nixed the variable was uneeded.